Blobs can now expand onto lattices and catwalks

ChangelingRain opened the pull request:
:cl: Joan
rscadd: Blobbernauts can now speak to overminds and other blobbernauts with :b
rscadd: Blobs can now expand onto lattices and catwalks.
rscdel: Blob expansion on space tiles with no supports is much slower.
tweak: Blob Sorium and Dark Matter now have less range on targets with bio protection.
tweak: Blob Sorium does slightly less damage.
/:cl:

Updates blob tips.
Blobs now consume walls that are on their tile.
Added a space between blob follow link and message.

ChangelingRain committed:
Blob Sorium and Dark Matter properly reduce range based on bio protection.
Updates blob tips.
Blobs now consume walls that are on their tile.
Added a space between blob follow link and message.

ChangelingRain commented on config/tips.txt:
```diff
@@ -133,12 +133,13 @@ As an Alien, the facehugger is by far your most powerful weapon because of its a
 As the Blob, remember that you can only expand onto tiles from cardinal directions, while the crew can attack you diagonally. Fight in narrow corridors or expand and widen across the width of the hallway to force confrontations where you can hit them!
 As the Blob, don't neglect the creation of factories. These create spores that carry your reagent and can chase crewmembers far further than you. Spores can also be rallied to swarm the crew and cause panic, and can even take over corpses to create much more dangerous blob zombies!
+As the Blob, you can expand by clicking, create strong blobs with ctrl-click, rally spores with middle-click, and remove blobs with alt-click.
+As the Blob, removing strong blobs, resource nodes, factories, and nodes will give you 2, 15, 25, and 25 resources back, respectively.
```
Yes, even the core shields. Are you sure enough of your power to use your defenses as resources?

Shadowlight213 replied:
Don't the core shields auto regen into strong blobs though? You can farm those for points.

WJohn commented:
Well then, this definitely makes attacking solar-based emitter platforms much easier than before.

ChangelingRain commented:
Fun fact; you can cut lattices and catwalks with a welding tool, which will remove them. This is a good way to cuck the blob.

Ultimate-Chimera commented:
:+1:

ChangelingRain committed:
Blobbernauts can now speak to other blobbernauts and overminds with :b
Blobbernauts and blob spores now have ask, exclaim, and yell speak emotes.
Blobbernauts and blob spores have a randomly-generated number assigned to their names.
